### PR TITLE
pse.tum.sexy WS20/21

### PR DESCRIPTION
**Pull request by jpbernius**

### CODE REVIEW

File: redirect/routes.php [hunk 1]
```diff
@@ -597,7 +596,8 @@ class Route {
         'complexity'     => 'comp',
         'sp-ge'          => 'ge-sp',
         'roomfinder'     => 'finder',
-        'patterns'       => 'pat',
+        'patterns'       => 'pse',
+        'pat'            => 'pat',
```
Comment (joschahenningsen) [inline redirect/routes.php:600]: This should probably point to pse as well right?

**Review by joschahenningsen — CHANGES_REQUESTED** [review]

File: redirect/routes.php (hunk 1 above)
Comment (jpbernius) [reply redirect/routes.php]: Good catch!

**Comment by joschahenningsen** [comment]
The pat index in the electives array (https://github.com/TUM-Dev/TUM.sexy/blob/ecb938f2dbf9923640ad2fe1b1bd8b73fd1a870d/redirect/routes.php#L719) needs to be updated as well I suppose.

**Comment by jpbernius** [comment]
All that nonsense to get rid of `pat.` :D

**Review by joschahenningsen — APPROVED** [review]

**Comment by joschahenningsen** [comment]
Looks good, thanks! :D

**Pull request merged**